chroma-manager/tests/framework/integration/installation_and_upgrade/run_tests

```diff
@@ -43,16 +57,31 @@ if [ ${PIPESTATUS[0]} != 0 ]; then
 fi
 
 # first fetch and install chroma 3.1.1.0
-BUILD_JOB=ieel
-BUILD_NUM=37
-IEEL_FROM_ARCHIVE=$(curl -s -k "${JENKINS_URL}job/$BUILD_JOB/$BUILD_NUM/api/xml?xpath=*/artifact/fileName&wrapper=foo" | sed -e 's/.*>\([i]\?ee[l]\?-[0-9\.][0-9\.]*.tar.gz\)<.*/\1/')
+BUILD_JOB=ieel-b3_1
+BUILD_NUM=427
+set +x # don't remove this line lest you leak confidential information
+. ~/auth.sh
+cat <<"EOF"
+++ curl -s -k 'https://jenkins-pull:********@jenkins-old.lotus.hpdd.lab.intel.com:8080/job/ieel-b3_1/427/api/xml?xpath=*/artifact/fileName&wrapper=foo'
```

Comment:
Doesn't matter at the moment, but worth thinking about what we do when jenkins-old goes away.

Reply:
Yup.  Trying to consider how this would be done then is one of the factors that has led to the procrastination of updating this test.